docs/locale/pt_BR/LC_MESSAGES/use-plugin-on-sonarqube.po

```diff
@@ -0,0 +1,163 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-2021, spotbugs community
+# This file is distributed under the same license as the spotbugs package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: spotbugs 4.4\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-10-22 15:35-0300\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: ../../use-plugin-on-sonarqube.rst:2
+msgid "Use SpotBugs Plugin on SonarQube"
+msgstr "Uso do SpotBugs Plugin no SonarQube"
+
+#: ../../use-plugin-on-sonarqube.rst:4
+msgid ""
+"`The SpotBugs SonarQube Plugin <https://github.com/spotbugs/sonar-"
+"findbugs>`_ uses major SpotBugs plugins such as `fb-contrib <http://fb-"
+"contrib.sourceforge.net/>`_ and `Find Security Bugs "
+"<http://h3xstream.github.io/find-sec-bugs/>`_. However, if you want to "
+"use another SpotBugs plugin, you need to build your own SonarQube plugin."
+" For detailed requirements on SonarQube plugins, see `the SonarQube "
+"official guidelines`_."
+msgstr ""
+"O `SpotBugs SonarQube Plugin  <https://github.com/spotbugs/sonar-"
+"findbugs>`_ usa os principais plugins do SpotBugs, como `fb-contrib <http://fb-"
+"contrib.sourceforge.net/>`_ and `Find Security Bugs "
+"<http://h3xstream.github.io/find-sec-bugs/>`_. No entanto, se quiser usar outro "
+"plugin, você precisa criar seu próprio plugin SonarQube. Para obter os requisitos "
+"detalhados dos plugins do SonarQube, consulte as diretrizes oficiais da SonarQube."
```

Comment:
[readthedocs reported](https://readthedocs.org/projects/spotbugs-pt-br/builds/15191127/) a warning as follows: `/home/docs/checkouts/readthedocs.org/user_builds/spotbugs-pt-br/checkouts/1796/docs/use-plugin-on-sonarqube.rst:4: WARNING: inconsistent references in translated message. original: ['`the SonarQube official guidelines`_'], translated: []`

We have a linked text `diretrizes oficiais do SonarQube` in at line 54, so we can wrap the same text in this line:

```suggestion
"detalhados dos plugins do SonarQube, consulte as `diretrizes oficiais do SonarQube`_."
```

If `da` is better than `do`, need to fix both of this line and line 54.

Reply:
Both ways are correct. For consistency with the rest of the page, we can use "do SonarQube" =)